dataservice/api/genomic_file/schemas.py

```diff
@@ -58,5 +58,7 @@ class Meta(BaseSchema.Meta, IndexdFileSchema.Meta):
             'api.sequencing_experiments',
             kf_id='<sequencing_experiment_id>'),
         'cavatica_task_genomic_files': ma.URLFor(
-            'api.cavatica_task_genomic_files_list', genomic_file_id='<kf_id>')
+            'api.cavatica_task_genomic_files_list', genomic_file_id='<kf_id>'),
+        'read_groups': ma.URLFor(
```

Comment:
Again, I'm not sure whether the link name should be `read_groups` or `read_group`. It depends on how the relationship should be defined.